### PR TITLE
feat: add moodboard gen with dall-e

### PR DESCRIPTION
**Pull request by clara-gdef**
Those are the first commits of the project, aiming at providing a MVP of the Weekly Theme Product.

2 main features here:
1. Random drawing of a theme from a given list
2. Mood board generation with an OpenAI call

Please let me know if the documentation, README.md or anything else can be improved.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,21 +15,32 @@ See the `pyproject.toml` file for the required packages.
...
+### Usage
+
+You can use this package by running the following command:
+
+```bash
+ poetry run python weekly_theme/src/weekly_theme/theme_moodboard.py
+```
+
 ## TODO
 - [ ] Add a connection to Google Keep to sync your weekly themes with a not you can modify from your phone
```
Comment (Louise-GdeFdeG) [inline README.md:37]: Small typo here `with a not you can...` -> `with a note you can ...`

File: README.md [hunk 2]
```diff
@@ -15,21 +15,32 @@ See the `pyproject.toml` file for the required packages.
...
+- [ ] Automate the draw so that it happens every Sunday at 12:00.
```
Comment (Louise-GdeFdeG) [inline README.md:38]: Yeaaaah

File: weekly_theme/src/weekly_theme/draw_theme.py [hunk 3]
```diff
@@ -1,18 +1,17 @@
-import ipdb
-import numpy as np
+import random
 
-from config import DATA_DIR, THEME_FILE, PASSED_THEMES_FILE
+from .config import DATA_DIR, THEME_FILE, PASSED_THEMES_FILE
```
Comment (Louise-GdeFdeG) [inline weekly_theme/src/weekly_theme/draw_theme.py:3]: I don't know this syntax: why do you need the dot before `config` for your imports?

File: weekly_theme/src/weekly_theme/draw_theme.py [hunk 4]
```diff
@@ -40,5 +39,4 @@ def _file_checks():
 
 
 if __name__ == "__main__":
-    with ipdb.launch_ipdb_on_exception():
-        main()
+    main()
```
Comment (Louise-GdeFdeG) [inline weekly_theme/src/weekly_theme/draw_theme.py:42]: RIP ipdb :skull:

**Review by Louise-GdeFdeG — APPROVED** [review]
Everything looks great! Just a few comments/questions, but nothing very important.  
Can't wait to see what's next :rocket:

File: weekly_theme/src/weekly_theme/draw_theme.py (hunk 3 above)
Comment (clara-gdef) [reply weekly_theme/src/weekly_theme/draw_theme.py:3]: so I've clearly mishandled my imports but I'm not great at Python packages so I hacked the code until it worked...

**Pull request merged**